### PR TITLE
Fix material metdata

### PR DESCRIPTION
**Pull request by makeclean**
This PR adds and updates tests for UWUW, previous to these commits metadata from the material objects in a material library were not propgated into the material library inserted into a DAGMC geometry.

### CODE REVIEW

File: uwuw/uwuw_preprocessor.cpp [hunk 1]
```diff
@@ -676,9 +677,9 @@ std::string name_concatenator::shift_and_increment(std::string name)
       name[6] = int_as_string[0];
       name[7] = int_as_string[1];
     }  else if(count >= 100 && count < 1000 ) {
-      name[6] = int_as_string[0];
-      name[7] = int_as_string[1];
-      name[8] = int_as_string[2];
+      name[5] = int_as_string[0];
```
Comment (pshriwise) [inline uwuw/uwuw_preprocessor.cpp:680]: Why the change in indices here?

File: CMakeLists.txt [hunk 2]
```diff
@@ -96,9 +96,7 @@ if (BUILD_FLUKA)
 endif (BUILD_FLUKA)
 
 # Include UWUW in dagmciface library
-if (BUILD_ANY)
-  add_subdirectory(uwuw)
-endif (BUILD_ANY)
+add_subdirectory(uwuw)
```
Comment (pshriwise) [inline CMakeLists.txt:99]: So the uwuw preprocessor is now part of the default build? As `make_watertight` and other tools are?

File: uwuw/uwuw_preprocessor.cpp (hunk 1 above)
Comment (makeclean) [reply uwuw/uwuw_preprocessor.cpp:680]: Because they were wrong, only picked up by the -e on the tests :)

File: CMakeLists.txt (hunk 2 above)
Comment (makeclean) [reply CMakeLists.txt:99]: Yeah, its useful without wanting to install the physics packages

File: uwuw/uwuw_preprocessor.cpp (hunk 1 above)
Comment (pshriwise) [reply uwuw/uwuw_preprocessor.cpp:680]: Stared at it for a bit. Got it.

File: CMakeLists.txt (hunk 2 above)
Comment (pshriwise) [reply CMakeLists.txt:99]: Sounds good to me.
